src/styles/globals.css

```diff
@@ -14,7 +14,7 @@ body:has(main[data-color-scheme="home"]) {
 
 body:has(main[data-color-scheme="oval"]) {
   color: var(--white);
-  --background-base: 0 0% 12%;
+  --background-base: 0 0% 13.8%;
```

Comment:
this might need to change based on browser and os??

Reply:
![image](https://github.com/UMAprotocol/uma.xyz/assets/4429761/587ca44d-67c2-4b22-88ef-91a299d0a5b6)
chrome/windows